plugins/fabrique/Chart.yaml

```
@@ -10,6 +10,10 @@ dependencies:
     repository: file://./charts/pg
     version: 1.161.4
     condition: pg.enabled
+  - name: maildev
+    repository: file://./charts/maildev
+    version: 1.136.2
```

```suggestion
    version: 1.136.3
```

c'est 1.136.3 dans maildev/Chart.yaml